### PR TITLE
🔨 Fix Upcoming Opening statistic mismatch

### LINKED ISSUE #1031: Upcoming opening statistic mismatch
The [statistics](https://www.figma.com/file/QSpueQlOQKGbiHig2harPy/Working-groups?node-id=1080%3A4835680) of the Working Group Opening should be:

*   Time To Begun
*   Opening Expected duration
*   Reward per x blocks (depending on WG) (might be a single number)
*   Open applicants (or minimal stake if there are no applicants/hiring limits in the API)

The footer should have: Created instead of Hired

### PR DESCRIPTION
**Pull request by thesan**
Closes #1031

### CODE REVIEW

**Comment by thesan** [comment]
@jodator I couldn't find a anything about the applicant limit, nor about the range of reward in the API. Did I miss something ?

**Comment by jodator** [comment]
> @jodator I couldn't find a anything about the applicant limit, nor about the range of reward in the API. Did I miss something ?

I think that the API might changed from the time of the wireframes/desings. I've crated a ticket to fix the application / hiring limit (#1110). For the reward range - I think that we can go with one value (not the range).

File: packages/ui/src/app/pages/WorkingGroups/UpcomingOpening.tsx [hunk 1]
```diff
@@ -74,10 +74,18 @@ export const UpcomingOpening = () => {
               </BadgeStatus>
             </BadgesRow>
             <Statistics>
-              <TokenValueStat title="Current budget" tooltipText="Lorem ipsum..." value={opening.budget} />
-              <DurationStatistics title="Opening Expected duration" value={opening.expectedEnding} />
+              <DurationStatistics title="Time to begun" value={opening.expectedStart} />
```
Comment (p-sad) [inline packages/ui/src/app/pages/WorkingGroups/UpcomingOpening.tsx:77]:
```suggestion
              <DurationStatistics title="Time to begin" value={opening.expectedStart} />
```
Looks like a strange typo

**Review by p-sad — APPROVED** [review]
Good overall, spotted just one typo/grammar error

**Pull request merged**